scripts/fuzz_shell.js

```diff
@@ -152,7 +152,15 @@ var imports = {
     // Throw an exception from JS.
     'throw': () => {
       throw 'some JS error';
-    }
+    },
+
+    // Table operations.
+    'table-get': (index) => {
+      return exports.table.get(index >>> 0);
```

Comment:
What's the reason for shifting the index by 0? (I googled it and it says it makes it a number, but I don't understand why it is necessary...)

Reply:
the `>>>` operator is an unsigned operation, so this forces it to be a 32-bit unsigned number. That is the same way that the wasm `table.get` operation treats the input, so this is done to match that.